csm/cli/support_bundle/support_bundle.py

```diff
@@ -189,14 +190,23 @@ async def bundle_status(command):
         :param command: Csm_cli Command Object :type: command
         :return: None
         """
-        bundle_id = command.options.get("bundle_id")
-        conf = GeneralConfig(Yaml(const.DATABASE_CLI_CONF).load())
-        db = DataBaseProvider(conf)
-        repo = SupportBundleRepository(db)
-        all_nodes_status = await repo.retrieve_all(bundle_id)
-        response = {"status": [each_status.to_primitive() for each_status in
+        try:
+            bundle_id = command.options.get("bundle_id", "")
+            conf = GeneralConfig(Yaml(const.DATABASE_CONF).load())
```

Comment:
Why **const.DATABASE_CLI_CONF** was changed to **const.DATABASE_CONF**? Looks like this will break cortxcli call.

Reply:
thanks corrected.